scanpy/preprocessing/_highly_variable_genes.py

```diff
@@ -97,90 +103,133 @@ def highly_variable_genes(
     if min_disp is None: min_disp = 0.5
     if min_mean is None: min_mean = 0.0125
     if max_mean is None: max_mean = 3
+    if max_disp is None: max_disp = np.inf
 
-    X = np.expm1(adata.X) if flavor == 'seurat' else adata.X
```

Comment:
These lines are actually not removed, but indented and moved below. It's very unfortunate that we still don't have smarted diff utils.